metadata.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * MetaDatalist Element
+ *
+ * @package     Joomla.Plugin
+ * @subpackage  Fabrik.list.metadata
+ * @copyright   Copyright (C) 2025 Jlowcode Org - All rights reserved.
+ * @license     GNU/GPL http://www.gnu.org/copyleft/gpl.html
+ */
+
+// No direct access
+defined('_JEXEC') or die('Restricted access');
+
+use Joomla\CMS\Factory;
```

Comment:
Adicione na linha debaixo 

```suggestion
use \Joomla\CMS\Uri\Uri;
```

Reply:
Adicionado!